keps/sig-node/2411-cri-container-log-rotation/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: CRI Container Log Rotation
+kep-number: 2411
+authors:
+  - "@umohnani8"
+owning-sig: sig-node
+participating-sigs:
+  - sig-instrumentation
+  - sig-storage
+status: implemented
+creation-date: 2018-01-25
+reviewers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+  - "@derekwaynecarr"
+prr-approvers:
+  - "@derekwaynecarr"
+  - "@dchen1107"
```

Comment:
same here, please update to a valid approver

Reply:
Fixed :)